config/configmaps/install-log-regexes-configmap.yaml

```diff
@@ -5,7 +5,13 @@ metadata:
   namespace: hive
 data:
   regexes: |
-    # AWS Specific:
+    # AWS Specific
+    # https://bugzilla.redhat.com/show_bug.cgi?id=1844320
+    - name: AWSUnableToFindMatchingRouteTable
+      searchRegexStrings:
+      - "Error: Unable to find matching route for Route Table.*CIDR block (0.0.0.0/0)"
```

Comment:
Those braces around subnet mask look like they would cause problems, should we just go with "Error: Unable to find matching route for Route Table"